libs/accounts/src/lib/accounts-table.tsx

```diff
@@ -1,170 +1,179 @@
+import type { CSSProperties } from 'react';
 import { forwardRef } from 'react';
 import type {
-  ColumnApi,
+  CellRendererSelectorResult,
   GroupCellRendererParams,
+  ICellRendererParams,
   ValueFormatterParams,
 } from 'ag-grid-community';
 import {
   PriceCell,
   addDecimalsFormatNumber,
   t,
-  addSummaryRows,
+  formatNumberPercentage,
 } from '@vegaprotocol/react-helpers';
-import type { SummaryRow } from '@vegaprotocol/react-helpers';
-import { AgGridDynamic as AgGrid } from '@vegaprotocol/ui-toolkit';
+import { Button, Intent } from '@vegaprotocol/ui-toolkit';
+import { AgGridDynamic as AgGrid, ProgressBar } from '@vegaprotocol/ui-toolkit';
 import { AgGridColumn } from 'ag-grid-react';
-import type { AgGridReact } from 'ag-grid-react';
+import type { AgGridReact, AgGridReactProps } from 'ag-grid-react';
 import type { AccountFieldsFragment } from './__generated__/Accounts';
 import { getId } from './accounts-data-provider';
 import { useAssetDetailsDialogStore } from '@vegaprotocol/assets';
-import type { AccountType } from '@vegaprotocol/types';
+import type { AccountFields } from './accounts-manager';
+import BigNumber from 'bignumber.js';
 import { AccountTypeMapping } from '@vegaprotocol/types';
+import type { AccountType } from '@vegaprotocol/types';
 
-interface AccountsTableProps {
-  data: AccountFieldsFragment[] | null;
+export interface ValueProps {
+  valueFormatted?: {
+    low: string;
+    high: string;
+    percentage: string;
+    value: number;
+    intent?: Intent;
+  };
 }
 
-interface AccountsTableValueFormatterParams extends ValueFormatterParams {
-  data: AccountFieldsFragment;
-}
+export const EmptyCell = () => '';
 
-export const getGroupId = (
-  data: AccountFieldsFragment & SummaryRow,
-  columnApi: ColumnApi
-) => {
-  if (data.__summaryRow) {
-    return null;
-  }
-  const sortColumnId = columnApi.getColumnState().find((c) => c.sort)?.colId;
-  switch (sortColumnId) {
-    case 'asset.symbol':
-      return data.asset.id;
-  }
-  return undefined;
+export const ProgressBarCell = ({ valueFormatted }: ValueProps) => {
```

Comment:
chore (blocking): This componet looks like a dupe of that used in the positions table, can we move this to react helpers grid?

Reply:
there are a few small differences but we can make it more configurable and move it.